dbt/dbt_project.yml

```diff
@@ -32,7 +32,4 @@ clean-targets:         # directories to be removed by `dbt clean`
 # as tables. These settings can be overridden in the individual model files
 # using the `{{ config(...) }}` macro.
 models:
-  my_new_project:
-      # Applies to all files under models/example/
-      example:
-          materialized: view
+  staging: view
```

Comment:
I'm working from dbt's suggestions of having the default materialization be views with the exception being ones that will be used for report queries (the fact and dim tables) and ones that are referenced more than once, in which case they will be tables. I have overridden the default in the config for the relevant model (and made a comment).